xml-occurrence-parsers/src/main/java/org/gbif/xml/occurrence/parser/parsing/extendedrecord/ExtendedRecordConverter.java

```diff
@@ -22,10 +22,12 @@ private ExtendedRecordConverter() {
   public static ExtendedRecord from(RawOccurrenceRecord rawRecord) {
 
     String recordId = rawRecord.getId();
+    // FIXME: check if this has to be done here or in the XmlFragmentParser or in both places
     if (Strings.isNullOrEmpty(recordId)) {
```

Comment:
can all this be removed since it is already done by XmlFragmenterParser

Reply:
finally removed since it's only used in this parser. All tests passed.